net/host.go

```diff
@@ -155,27 +146,56 @@ func NewHost(cfg *Config) (*host, error) {
 		return nil, fmt.Errorf("failed to format bootnodes: %w", err)
 	}
 
+	if len(bns) > 0 {
+		opts = append(opts, libp2p.EnableAutoRelay(autorelay.WithStaticRelays(bns)))
```

Comment:
I assume this requires the bootnodes to have relay enabled?

Reply:
Yes, it does.  Relay is enabled by default in libp2p.  We were previously disabling it.